pom.xml

```diff
@@ -123,29 +128,65 @@
         </dependency>
 
         <!-- test dependencies -->
+
+        <!-- assertions library -->
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <!-- use 2.9.1 for Java 7 projects -->
+            <version>2.9.1</version>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- junit 5 -->
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
+            <artifactId>junit-jupiter</artifactId>
             <version>${junit5.version}</version>
             <scope>test</scope>
         </dependency>
+
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
+            <artifactId>junit-jupiter-api</artifactId>
             <version>${junit5.version}</version>
             <scope>test</scope>
         </dependency>
+
```

Comment:
There is a JUnit5 cucumber engine, but it isn't ready for production use. Enabling `junit-vintage-engine` lets us continue to use the JUnit4 cucumber tooling.